kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/container/orchestration/ImageConfiguration.java

```diff
@@ -0,0 +1,145 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Eurotech and/or its affiliates and others
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *  Eurotech
+ *******************************************************************************/
+
+package org.eclipse.kura.container.orchestration;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.Objects;
+import java.util.Optional;
+
+import org.osgi.annotation.versioning.ProviderType;
+
+/**
+ * Object which represents a image configuration used to request the generation
+ * of a new image instance
+ *
+ * @noimplement This interface is not intended to be implemented by clients.
+ * @since 2.4
+ *
+ */
+@ProviderType
+public class ImageConfiguration {
+
+    private String imageName;
+    private String imageTag;
+    private Optional<RegistryCredentials> registryCredentials;
+    private int imageDownloadTimeoutSeconds = 500;
+
+    private ImageConfiguration() {
+    }
+
+    /***
```

Comment:
Also here there were comments in the previous PR related to the format of the javadoc: /** instead of /***